android/manifest

```diff
@@ -2,7 +2,7 @@
 # this is your module manifest and used by Titanium
 # during compilation, packaging, distribution, etc.
 #
-version: 4.0.0
+version: 5.0.1
```

Comment:
I think this can be `5.0.0` since the `package.json` is indepdendent of this

Reply:
Done